README.md

```diff
@@ -61,6 +61,7 @@ Various CMake options are available to customize the client/server SDK builds.
 | `LD_BUILD_CONTRACT_TESTS` | Whether the contract test service (used in CI) is built.                                                                                          | Off                                                    | `BUILD_TESTING`                           |
 | `LD_BUILD_EXAMPLES`       | Whether example apps (hello world) are built.                                                                                                     | On                                                     | N/A                                       |
 | `LD_BUILD_SHARED_LIBS`    | Whether the SDKs are built as static or shared libraries.                                                                                         | Off  (static lib)                                      | N/A                                       |
+| `LD_BUILD_EXPORT_SYMBOLS` | Whether to export C++ symbols in shared libraries. C++ does not have an ABI, so only use this feature if you understand the risk and requirements. A mismatch in ABI could cause crashes or other unexpected behaviors.| Off  (hidden) | `LD_BUILD_SHARED_LIBS` |
```

Comment:
```suggestion
| `LD_BUILD_EXPORT_ALL_SYMBOLS` | Whether to export all symbols in shared libraries. By default, only C API symbols are exported because C++ does not have an ABI. Only use this feature if you understand the risk and requirements. A mismatch in ABI could cause crashes or other unexpected behaviors.| Off  (hidden) | `LD_BUILD_SHARED_LIBS` |
```